### ISSUE TITLE
All my entities renamed with _2 after filename on 0.0.19 from 0.0.18

### ISSUE THREAD

**Issue by filmgarage** [StyraHem/ShellyForHASS#57]
After installing all my HA entities have "_2" added to their id.

I fixed all of them manually before upgrading to version 0.19
After installing 0.19 they were back with this addition.

Just cleaned them all out again... Be warned!

:-)

**Comment by hakana** [comment]
Did you install 0.0.18 first?

**Comment by filmgarage** [comment]
Indeed I did!

**Comment by hakana** [comment]
Sorry for that! I changed back the unique_id as it was before 0.0.18 so that is the reason you got it twice. Hopefully not so many installed 0.0.18 before I removed it and replace it with release 0.0.19.SO for every one that upgrade from 0.0.15 it will work without any issues.

**Comment by filmgarage** [comment]
I will give you a run-down of what I did, maybe it helps...

Where I came from:
All the entity_id's were already altered to the shellies before I could set them using the Shelly plugin. I had custumized them one by one. Very cumbersome. I was happy with the new settings of ShellyforHASS. Couldn't wait to upgrade...

What happened after I upgraded to 0.18 was the following: 
After the reboot all the names had the extra '_2'. That should not have been a surprise. Because the names were already reserved in HA... So I deleted all the names with the added '_2' as well as the original customised ones. After a reboot all was good.

But after the update I ended up with new '_2' suffixes. Probably the names were again reserved for the 0.18 entries???

Anyway, I am getting used to cleaning it up and get faster at it! ;-)
